Formula/mvtools.rb

```diff
@@ -24,6 +24,9 @@ class Mvtools < Formula
   depends_on "fftw"
   depends_on "vapoursynth"
 
+  # Fixes build issues on arm
```

Comment:
Worth cross-referencing the pull request in case anyone in the future is looking for it:

```suggestion
  # Fixes build issues on arm
  # https://github.com/dubhater/vapoursynth-mvtools/pull/55
```